app/helpers/ama_layout_navigation_helper.rb

```diff
@@ -0,0 +1,28 @@
+module AmaLayoutNavigationHelper
```

Comment:
Actually... it looks like other helpers are following the same  naming scheme. Let's not worry about it.